src/generate.jl

```diff
@@ -216,6 +216,10 @@ function gen_readme(pkg_dir::AbstractString, t::Template)
             "\n",
         )
     end
+    if haskey(t.plugins, Citation) && t.plugins[Citation].readme_section
+        text *= "\n## Citing\n\nSee `CITATION.bib` for the relevant reference(s)\n"
```

Comment:
```suggestion
        text *= "\n## Citing\n\nSee `CITATION.bib` for the relevant reference(s).\n"
```


Punctuation please 🙂

Reply:
fixed in last commit